### PR TITLE
Feature/data attributes configuration endpoint

### LINKED ISSUE #387: Exposing config for fields to admin and element SDK
**Is your feature request related to a problem? Please describe.**
None.

**Describe the solution you'd like**
Expose fields to configure fields on admin and showing them on front-end. We should have a PUT / GET.

**Describe alternatives you've considered**
None.

**Additional context**
None.

### PR DESCRIPTION
**Pull request by segoddnja**
create endpoints to fetch and update
create migration with example config
create test suite
implement basic config validation

closes #387

### CODE REVIEW

File: cg/src/domains/management/data/data.service.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const KEY = 'SUBJECT_DATA_ATTRIBUTES_CONFIG';
```
[the remark this answers is not in the document]
Comment (segoddnja) [reply cg/src/domains/management/data/data.service.js]: I disagree. This const is for internal usage only, what is the sense to export it? Only to use it in seeds/migration and tests? Exposing something outside only for testing purpose is a bad practice.

File: cg/migrations/20180815171916_data-attributes-configuration.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const KEY = 'SUBJECT_DATA_ATTRIBUTES_CONFIG';
```
[the remark this answers is not in the document]
Comment (segoddnja) [reply cg/migrations/20180815171916_data-attributes-configuration.js]: See my comment below

File: cg/src/domains/management/data/data.service.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (fjcero) [reply cg/src/domains/management/data/data.service.js]: Ok, if we are using it only here makes sense to keep it that way.

File: cg/seeds/example_data_attributes_config.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+exports.seed = async function(knex, Promise) {
```
Comment (sindelio) [inline cg/seeds/example_data_attributes_config.js]: What will this seed be used for?

File: cg/src/domains/management/management.routes.js [hunk 4]
```diff
@@ -146,6 +151,17 @@ module.exports = app => {
     asyncHandler(async (req, res) => usersController.updatePassword(req, res))
   );
 
+  router.put(
```
Comment (sindelio) [inline cg/src/domains/management/management.routes.js]: We're mixing REST here, it makes no sense to start doing it now. Either we change everything or keep it as it was before. Please change these endpoints

File: cg/seeds/example_data_attributes_config.js (hunk 3 above)
Comment (fjcero) [reply cg/seeds/example_data_attributes_config.js]: For creating a default configuration in the Controller side, will be used mainly for demo and elements sdk

Comment (sindelio) [reply cg/seeds/example_data_attributes_config.js]: Oks

File: cg/src/domains/management/management.routes.js (hunk 4 above)
Comment (segoddnja) [reply cg/src/domains/management/management.routes.js]: Well, to some extent this makes sense, although I do not like and idea to do something wrong just to be consistent with something.

**Pull request merged**